CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
 #### Features
+- Introduced new `ViewOptionsCustomization` options that allows showing/hiding of various `NavigationView` subviews. [#6506](https://github.com/mapbox/mapbox-navigation-android/pull/6506)
```

Comment:
Could you clean up the changelog to only include the relevant change? To make sure that this plays well with the release tool.
cc @SevaZhukov